containers/podman-desktop
# feat: use podman v5 installer by default

### What does this PR do?
in 1.9 the v5 installer was experimental, now it's the default choice

Also updates the airgapped archive to include the v5 one and not the v4

### Screenshot / video of UI



### What issues does this PR fix or reference?

fixes https://github.com/containers/podman-desktop/issues/6360

### How to test this PR?

Try airgapped installer (`AIRGAP_DOWNLOAD=true yarn compile:current`)

Try default path, having a v4 or v5 (not 5.0.1 which is the latest), it should always report to update to v5
